### ISSUE TITLE
Default for empty alt text should be an empty attribute

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#105]
## Description

I think I saw this on last year's website, but all images without alt text get the string "Provide alt text," which ends up announcing out to screen reader users. If possible (maybe right before we launch), I'd recommend replacing that with just an empty alt tag `""` which will make it so the image announces nothing 🙇 

<img width="528" alt="Screen Shot 2022-09-23 at 6 32 37 PM" src="https://user-images.githubusercontent.com/14154792/192065238-c97db304-90ab-408e-aaef-e49c53cb97f6.png">

should be

<img width="489" alt="Screen Shot 2022-09-23 at 6 32 54 PM" src="https://user-images.githubusercontent.com/14154792/192065260-4b66cee6-56de-42d2-a856-776cbd152118.png">
